configs/laramie_bmi_config_smc_coupler.txt

```diff
@@ -0,0 +1,11 @@
+soil_params.smcmax=0.439
```

Comment:
Same question here: Do we need to add units to this config file similar to the cfe config? [units]

Reply:
done